bmaptools/TransRead.py

```diff
@@ -55,7 +56,44 @@
 # A list of supported compression types
 SUPPORTED_COMPRESSION_TYPES = ('bz2', 'gz', 'xz', 'lzo', 'lz4', 'tar.gz',
                                'tar.bz2', 'tar.xz', 'tar.lzo', 'tar.lz4',
-                               'zip')
+                               'zip', 'tar')
+
+# Allow some file extensions to be recognised as aliases of others
+EXTENSION_ALIASES = {
+    '.tbz2': '.tar.bz2',
+    '.tbz':  '.tar.bz2',
+    '.tb2':  '.tar.bz2',
+    '.tgz':  '.tar.gz',
+    '.txz':  '.tar.xz',
+    '.tzo':  '.tar.lzo',
+    '.tlz4': '.tar.lz4',
+    '.gzip': '.gz',
+}
+
+# These need to be checked *before* those in EXTENSION_ALIASES
+NESTED_EXTENSION_ALIASES = {
+    '.tar.gzip': '.tar.gz',
+}
+
```

Comment:
I think it would be cleaner and easier to read if we had a signle mapping data structure, not 2 of them that have to be used in the right order. Something like this:

EXTENSION_ALIASES = {
    (".tbz2", ".tbz", ".tb2") : ".tar.bz2",
    (".tgz", ".tar.gzip") : ".tar.gz",
... and so on
}

Note, the checking order will be the order in the tuple. Worth mentioning in a comment.